beetmoverscript/test/test_utils.py

```diff
@@ -441,38 +441,40 @@ def test_beetmover_template_args_generation_release_is_jar(context):
         'artifact_id': 'fenix-megazord-forUnitTests',
         'is_jar': True,
         'template_key': 'maven_Fake',
-        'version': '0.26.20990205110000',
+        'version': '0.26.0',
     }
 
     template_args = generate_beetmover_template_args(context)
     assert template_args == expected_template_args
 
 
-@pytest.mark.parametrize('branch, version, artifact_id, build_id, expected_version, raises', ((
-    'mozilla-central', '63.0a1', 'geckoview-nightly-x86', '20181231120000', '63.0.20181231120000', False,
+@pytest.mark.parametrize('product, branch, version, artifact_id, build_id, expected_version, raises', ((
+    'geckoview', 'mozilla-central', '63.0a1', 'geckoview-nightly-x86', '20181231120000', '63.0.20181231120000', False,
 ), (
-    'mozilla-beta', '63.0b2', 'geckoview-beta-armeabi-v7a', '20181231120000', '63.0.20181231120000', False,
+    'geckoview', 'mozilla-beta', '63.0b2', 'geckoview-beta-armeabi-v7a', '20181231120000', '63.0.20181231120000', False,
 ), (
-    'mozilla-release', '63.0', 'geckoview-arm64-v8a', '20181231120000', '63.0.20181231120000', False,
+    'geckoview', 'mozilla-release', '63.0', 'geckoview-arm64-v8a', '20181231120000', '63.0.20181231120000', False,
 ), (
-    '', '0.25.1', 'browser-session', None, None, True,
+    'components', '', '0.25.1', 'browser-session', None, '0.25.1', False,
 ), (
-    '', '1.0', 'browser-session', None, None, True,
+    'components', '', '1.0.0', 'browser-session', None, '1.0.0', False,
+), (
+    'components', '', '1.0', 'browser-session', None, None, True,
```

Comment:
Maybe in a follow-up: copy-paste this line with `product == appservices` to ensure this won't break whenever we'll switch to `1.0.0` there as well?